Sources/Commands/Options.swift

```diff
@@ -328,6 +328,15 @@ public struct SwiftToolOptions: ParsableArguments {
         help: "Specify the .netrc file path.",
         completion: .file())
     var netrcFilePath: AbsolutePath?
+    
+#if canImport(Security)
+    /// Whether to use keychain for authenticating with remote servers
+    /// when downloading binary artifacts or communicating with a registry.
+    @Flag(inversion: .prefixedNo,
+          exclusivity: .exclusive,
+          help: "Search credentials in OS keychain")
```

Comment:
I'm with you that the term "keychain" should be qualified in some way. "OS" doesn't feel quite right, but I'm struggling to come up with something better. Maybe "system"?

```suggestion
          help: "Search credentials in system keychain")
```

Reply:
Maybe "macOS keychain"? The CLI won't really be available anywhere else where the keychain is supported, right?

Reply:
Updating to "macOS keychain"